### PR TITLE
Document KubeVirt VMI update automation

### PR DESCRIPTION
**Pull request by davidvossel**
We have a new feature that coordinates updating VMIs which has landed in kubevirt that will in the upcoming v0.39.0 release. https://github.com/kubevirt/kubevirt/pull/4686

### CODE REVIEW

File: docs/operations/updating_and_deletion.md [hunk 1]
```diff
@@ -43,6 +43,99 @@ need to update the `virt-operator` first, and then proceed to update
...
+
+The least disruptive way to update VMI workloads is to use LiveMigrate. Any
+VMI workload that is not live migratable will be left untouched. If live
+migration is not enabled in the cluster, then the only option available for
+virt-operator managed VMI updates is the Evict method.
+
+
+**Example: Enabling VMI workload updates via LiveMigration**
+
+```
```
Comment (mazzystr) [inline docs/operations/updating_and_deletion.md:81]: please update to ```console

File: docs/operations/updating_and_deletion.md [hunk 2]
```diff
@@ -43,6 +43,99 @@ need to update the `virt-operator` first, and then proceed to update
...
+      - LiveMigrate
+```
+
+**Example: Enabling VMI workload updates via Evict with batch tunings**
+
+The batch tunings allow configuring how quickly VMI's are evicted. In large
+clusters, it's desirable to ensure that VMI's are evicted in batches in order
+to distribute load.
+
+```
```
Comment (mazzystr) [inline docs/operations/updating_and_deletion.md:100]: please update to ```console

File: docs/operations/updating_and_deletion.md [hunk 3]
```diff
@@ -43,6 +43,99 @@ need to update the `virt-operator` first, and then proceed to update
...
+
+
+**Example: Enabling VMI workload updates with both LiveMigrate and Evict**
+
+When both LiveMigrate and Evict are specified, then any workloads which are
+live migratable will be guaranteed to be live migrated. Only workloads which
+are not live migratable will be evicted.
+
+
+```
```
Comment (mazzystr) [inline docs/operations/updating_and_deletion.md:123]: please update to ```console

**Review by mazzystr — CHANGES_REQUESTED** [review]

**Comment by mazzystr** [comment]
@davidvossel  Fix those nits so they don't show up in lintting and do a rebase then this pr is +1

**Comment by davidvossel** [comment]
@mazzystr fixed

**Pull request merged**